SimG4CMS/Calo/interface/HCalSD.h

```diff
@@ -14,6 +14,7 @@
 #include "SimG4CMS/Calo/interface/HFShowerPMT.h"
 #include "SimG4CMS/Calo/interface/HFShowerFibreBundle.h"
 #include "SimG4CMS/Calo/interface/HcalNumberingScheme.h"
+#include "SimG4CMS/Calo/interface/HcalTestNS.h"
```

Comment:
Hi @bsunanda - So I've answered the questions I said I had yesterday by looking through more of the code. It seems suboptimal to have buried test code deep within the sensitive detector implementation, but its not obvious to me that this can be avoided. So, thats good for now.

I have a few fix requests in the following comments.